keytransform/keytransform.go

```diff
@@ -224,6 +226,23 @@ func (d *Datastore) Batch(ctx context.Context) (ds.Batch, error) {
 	}, nil
 }
 
+func (d *Datastore) NewTransaction(ctx context.Context, readOnly bool) (ds.Txn, error) {
+	tds, ok := d.child.(ds.TxnDatastore)
+	if !ok {
+		return nil, fmt.Errorf("txn not supported")
```

Comment:
Prefer `errors.New` when no formatting. Also want to avoid abbreviations in message.
```suggestion
		return nil, errors.New("transaction not supported")
```

Reply:
de4fd81